docs/MODULES.md

```diff
@@ -247,6 +248,38 @@ This module is used for decoding a QR in image (if present).
            .addSteps('decode-qr',options)
            .run()
 ```
+
+## dither-module
+
+This module approximates a color from a mixture of other colors when the required color is not available, creating illusions of the color that is not present actually.
```

Comment:
Can you link to the Wikipedia page? You can do that by doing this.
```
[Wikipedia](https://wikipedia.com/link-to-page)
```

The link will be shown like [this](https://wikipedia.com)

The link to the Wikipedia page will be available in the info.json file.

Reply:
https://wikipedia.com/publiclab/image-sequencer/blob/main/docs/MODULES.md#white-balance-module is this kind of link that I've to add also where should I add the link?

Reply:
What I meant was the link about the Wikipedia article about what dither is. Please add a link to it

Reply:
Please add the Wikipedia link